scripts/fetch-poky.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+git clone --depth 1 --filter=blob:none --sparse https://github.com/yoctoproject/poky.git
+cd poky
+git fetch --tags
+TMP_TAG=$(git tag | grep "yocto-" | tail -n 1) # There is a tag: yocto_1.5_M5.rc8 which will take the tail, thus adding a hyphen
+LASTEST_RELEASE=$(git show $TMP_TAG | grep commit | sed "s/^commit //")
+LINK="https://downloads.yoctoproject.org/releases/yocto/$TMP_TAG/poky-$LASTEST_RELEASE.tar.bz2"
```

Comment:
Maybe it would be simpler to git clone poky and copy how fetch-docs work? I used tar.bz2 because it was faster to fetch. But if it's not significantly slower (>2min) to fetch through git, it might result in a more uniform code base.
This is just a question, not sure we want to do it.

Reply:
I was thinking about this too. Check the new commit.